src/tab/tabs/nav.jsx

```diff
@@ -424,21 +527,40 @@ class Nav extends React.Component {
 
         const containerCls = classnames({
             [`${prefix}tabs-nav-container`]: true,
-            [`${prefix}tabs-nav-container-scrolling`]: showNextPrev
+            [`${prefix}tabs-nav-container-scrolling`]: showNextPrev,
         });
 
         const navCls = `${prefix}tabs-nav`;
         const tabList = this.renderTabList(this.props);
 
         const container = (
             <div className={containerCls} key="nav-container">
-                <div className={`${prefix}tabs-nav-wrap`} ref={this.wrapperRefHandler}>
+                <div
+                    className={`${prefix}tabs-nav-wrap`}
+                    ref={this.wrapperRefHandler}
+                >
                     <div className={`${prefix}tabs-nav-scroll`}>
-                        {
-                            animation ?
-                                <Animate role="tablist" aria-multiselectable={false} component="ul" className={navCls} animation={navCls} singleMode={false} ref={this.navRefHandler}>{tabList}</Animate> :
-                                <ul className={navCls} ref={this.navRefHandler}>{tabList}</ul>
-                        }
+                        {animation ? (
+                            <Animate
+                                role="tablist"
+                                aria-multiselectable={false}
+                                component="ul"
+                                className={navCls}
+                                animation={navCls}
+                                singleMode={false}
+                                ref={this.navRefHandler}
+                            >
+                                {tabList}
+                            </Animate>
+                        ) : (
+                            <ul
+                                role="tablist"
```

Comment:
This line is the only real change, I mean add `role="tablist"`, others are all prettier thing.